CMakeLists.txt

```diff
@@ -87,7 +87,7 @@ option(ORC_PACKAGE_KIND
 
 option(ORC_ENABLE_CLANG_TOOLS
     "Enable Clang tools"
-    ON)
+    OFF)
```

Comment:
Hi, @wgtmac .

May I ask why we need to disable this for all build?
I guess `ORC_PACKAGE_KIND` has the information of `conan`.

Can we narrow down our change into the specific `conan` build?

Reply:
`ORC_ENABLE_CLANG_TOOLS` was required by the previous GitHub workflow for clang-tidy and clang-format enforcement. Now it is replaced by the `cpp-linter-action` and is no longer needed. At the same time, it defines custom targets like `check-clang-tidy` and `check-format` which may have name conflict with other projects (like Apache Arrow). In many build environments (like Conan or Github CIs) usually these tools are missing and result in annoying warning messages. So I think it makes sense to disable it by default.

Reply:
It sounds valid but independent from this PR. Maybe, could you handle them seperately next time please?

Reply:
BTW, do you think we need Apache ORC 2.1.1 for downstreams to deliver this? I'm wondering if this is the last one to help them.

Reply:
I'll keep it in mind next time :)

> BTW, do you think we need Apache ORC 2.1.1 for downstreams to deliver this? I'm wondering if this is the last one to help them.

I don't think we need it yet and we can still keep the release cadence for now. Let's wait for feedback from the Conan community. But releasing 2.1.1 is helpful anyway.